app/assets/v2/css/gitcoin.css

```diff
@@ -315,3 +315,161 @@ div.button-pink {
   /* IE 10+ */
   color: lightgray;
 }
+
+.ios { 
```

Comment:
Worth moving this to it's own file rather than keeping it in `gitcoin.css`?

Reply:
Yup, was thinking this as well. 

Moved all the ios into its own file.